### DISCUSSION TITLE
Is there a way to update a specific yaml metadata value?

### DISCUSSION THREAD

**Discussion by greg-fox** [chhoumann/quickadd#432] [Q&A] [answered]
In my daily note metadata I have a place I want to record the number of hours I worked for that day. Currently I am manually adding the value, but would much rather be able to use something like QuickAdd so I can update it from anywhere in my vault.

I've seen this in the docs: https://quickadd.obsidian.guide/docs/Examples/Macro_ChangePropertyInDailyNotes
but I cannot figure out what to do with that snippet, nor how to invoke it.

**Comment by chhoumann** [comment, marked as answer]
Hi @greg-fox 

The macro uses a script. Here's a guide to installing and using scripts:
https://youtu.be/gYK3VDQsZJo?t=1730

**Reply by greg-fox** [reply]
Aha. I had seen that video, but did not connect everything together until now. Thank you so much for the response!

**Reply by chhoumann** [reply]
Happy to help!

**Comment by chhoumann** [comment]
Closing this up since it was resolved. One update for anyone landing here now: these days the cleanest way to update a frontmatter property from a QuickAdd user script is Obsidian's own API:

```js
await app.fileManager.processFrontMatter(file, (fm) => {
  fm["hours worked"] = value;
});
```
